src/svg/plot-box.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?> <!-- Generator: IcoMoon.io --> <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"> <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="24" height="16" viewBox="0 0 24 16" enable-background="new 0 0.028 24 16" xml:space="preserve" fill="#666666"> <g><path d="M 21.004,2.009l0,4.491 L 18,6.5 l0-3.5 L 18,2.359 L 18,0 L 8.578,0 l0,0.001 L 6,0.001 l0,6.499 L 3.011,6.5 L 3.011,2.009 L0,2.009 l0,11.991 l 3.011,0 l0-4.5 L 6,9.5 l0,6.5 l 2.5,0 l 0.511,0 L 18,16 l0-2.812 l0-0.188 l0-3.5 l 3.004,0 l0,4.5 L 24,14 L 24,2.009 L 21.004,2.009 z M 9.011,13l0-10 L 15,3 l0,10 L 9.011,13 z"></path></g></svg>
```

Comment:
some of these codelines require a newline at the end

Reply:
I'm curious as to why? these are not used in the end, they are stripped and modified by the `generate.js` file, unless I am missing something :)

Reply:
https://robots.thoughtbot.com/no-newline-at-end-of-file

Reply:
in the grand scheme of things it likely won't matter. However if I wanted to chew through these svg files using linux tools like `cat awk sed` etc concatenating the files before sending them through a line parser will probably fail. It's a hygiene and best practices thing rather than a immediately practical one.

Reply:
Okay, cool. I _think_ I added them. I just added a return after each.. If there is better or more correct way to do it, I'd love to know!

Reply:
Thanks! That is all that is needed. While there is ambiguity between `\r` `\n` and `\r\n` most tooling out there knows how to handle these cases.